be/src/service/doris_main.cpp

```diff
@@ -266,6 +266,10 @@ int main(int argc, char** argv) {
 #if defined(LEAK_SANITIZER)
         __lsan_do_leak_check();
 #endif
+
+#if !defined(ADDRESS_SANITIZER) && !defined(LEAK_SANITIZER) && !defined(THREAD_SANITIZER)
+        doris::MemInfo::refresh_current_mem();
+#endif
         sleep(10);
```

Comment:
10s is too long. 
When I optimize the memory allocation later, I plan to manually call it in other places such as HashTable / ChunkAllocator.

Reply:
10ms